app.js

```diff
@@ -48,14 +48,14 @@ if (!location) {
       latitude,
       longitude,
       units,
-      (err, { description, temp, feelsLike, tempScale } = {}) => {
+      (err, { description, temp, feelsLike, tempScale, windSpeed, windDirection } = {}) => {
         if (err) {
           return spinner.fail(err);
         }
         spinner.succeed(chalk.underline(location));
         console.log(
           chalk.cyanBright(
-            `${description}. It is currently ${temp}${tempScale}, it feels like ${feelsLike}${tempScale}.`
+            `${description}. It is currently ${temp}${tempScale}, it feels like ${feelsLike}${tempScale}. The wind is ${windDirection} at ${windSpeed} mph.` 
           )
```

Comment:
Not handling units, check the weatherstack documentation

Reply:
I will double check the weather stack docs on handling units. Thank you!

Reply:
we are already handling units, for metric and scientific Wind Speed/Visibility: Kilometers/Hour for Fahrenheit Wind Speed/Visibility: Miles/Hour. I would just do something similar to `getTemperatureScale()` to handle the speed